scripts/globus/generate_web_server_credentials.py

```diff
@@ -40,7 +41,7 @@
 DOMAIN = os.getenv("DATAFED_DOMAIN", default_DOMAIN)
 if len(DOMAIN) == 0:
     DOMAIN = default_DOMAIN
-REDIRECT_PATH = os.path.join("https://", DOMAIN, "ui/authn")
+REDIRECT_PATH = urllib.parse.urljoin(f"https://{DOMAIN}","ui/authn")
```

Comment:
Honestly, I think would be simpler and fine. urllib.parse module is not really needed unless you have two variables you are combing and they are parts of path, but this is for the most part hard coded. 

```suggestion
REDIRECT_PATH = f"https://{DOMAIN}/ui/authn"
```